modules/auxiliary/gather/ip_geolocate.rb

```diff
@@ -0,0 +1,103 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
Correct me if I'm wrong, but I don't think these `require` statements are necessary. I know not the first, and I'm pretty sure not the second.